lms/djangoapps/teams/static/teams/js/views/teams.js

```diff
@@ -4,34 +4,45 @@
         'backbone',
         'teams/js/views/team_card',
         'common/js/components/views/paginated_view',
-        'teams/js/views/team_actions'
-    ], function (Backbone, TeamCardView, PaginatedView, TeamActionsView) {
+        'text!teams/templates/team-actions.underscore'
+    ], function (Backbone, TeamCardView, PaginatedView, teamActionsTemplate) {
         var TeamsView = PaginatedView.extend({
             type: 'teams',
 
+            events: {
+                'click a.browse-teams': 'browseTeams',
+                'click a.search-team-descriptions': 'searchTeamDescriptions',
+                'click a.create-team': 'showCreateTeamForm'
+            },
+
             initialize: function (options) {
                 this.topic = options.topic;
+                this.teamMemberships = options.teamMemberships;
+                this.teamParams = options.teamParams;
                 this.itemViewClass = TeamCardView.extend({
                     router: options.router,
                     topic: options.topic,
                     maxTeamSize: options.maxTeamSize,
                     countries: this.selectorOptionsArrayToHashWithBlank(options.teamParams.countries),
-                    languages: this.selectorOptionsArrayToHashWithBlank(options.teamParams.languages),
+                    languages: this.selectorOptionsArrayToHashWithBlank(options.teamParams.languages)
                 });
                 PaginatedView.prototype.initialize.call(this);
-                this.teamParams = options.teamParams;
-                this.showActions = options.showActions;
             },
 
             render: function () {
                 PaginatedView.prototype.render.call(this);
 
-                if (this.showActions === true) {
```

Comment:
Thanks @symbolist. I did a search and didn't see this, but I guess that's because it defaults to false. I'll add a Jasmine test so that it doesn't accidentally break in the future.